playbooks/vars/maas-poller.yml

```diff
@@ -0,0 +1,21 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# maas_poller_ping_check_count: Number of times to try ping check in private
+#                               network monitoring case. Must be integer
+#                               1-15 inclusive.
+private_ping_check_count: 6
+private_ssh_port: 22
```

Comment:
Fixed in the latest push. Added a small comment explanation for private_ssh_port and updated the name in the comment on the ping_check one.